### PR TITLE
[ESD-10528] fix: keyword mapping in client page templates

### PR DESCRIPTION
**Pull request by shushen**
## ✏️ Changes

Fix keyword mapping in client page templates

## 🔗 References

- #291 
- https://auth0team.atlassian.net/browse/ESD-10528

## 🎯 Testing

✅ This change has unit test coverage

🚫 This change has integration test coverage

🚫 This change has been tested for performance

### CODE REVIEW

File: src/context/yaml/handlers/clients.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ async function parse(context) {
           const htmlFileName = path.join(clientsFolder, client.custom_login_page);
 
           if (isFile(htmlFileName)) {
-            client.custom_login_page = loadFile(htmlFileName);
+            client.custom_login_page = context.loadFile(htmlFileName);
```
Comment (shawnmclean) [inline src/context/yaml/handlers/clients.js:25]: wheres `loadfile` set on `context`? Was it already there?

**Review by shawnmclean — APPROVED** [review]

File: src/context/yaml/handlers/clients.js (hunk 1 above)
Comment (shushen) [reply src/context/yaml/handlers/clients.js:25]: Yep: https://github.com/auth0/auth0-deploy-cli/blob/9b44071c5f01287c26a94a189ad15e1148e5fd3c/src/context/yaml/index.js#L36

**Comment by mmaddex** [comment]
@shushen Thanks for the quick response on this. Any idea when it will be worked into a release?

**Comment by shushen** [comment]
@mmaddex Sorry for the delay, was trying to bundle with an upcoming feature but it seems going to take a bit longer. I'm releasing v5.3.2 for this bug fix (#299).

**Comment by shushen** [comment]
@mmaddex v5.3.2 has been released https://www.npmjs.com/package/auth0-deploy-cli/v/5.3.2

**Pull request merged**